scaling_performance/network_optimization.adoc

```diff
@@ -114,18 +114,19 @@ in latency tests.
 {product-title} provides IP address management for both pods and services. The
 default values allow for:
 
-- Maximum cluster size of 1024 nodes
-- Each of the 1024 nodes has a /23 allocated to it (510 usable IPs for pods)
-- Provides 65,536 IP addresses for services.
+- A maximum cluster size of 1024 nodes
+- Each of the 1024 nodes having a /23 allocated to it (510 usable IPs for pods)
+- Around 65,536 IP addresses for services
 
-Under most circumstances, these networks cannot be changed after deployment.  Thus it is important to plan ahead for growth.
+Under most circumstances, these networks cannot be changed after deployment.
+So, planning ahead for growth is important.
 
-Restrictions for resizing networks are document here:
+Restrictions for resizing networks are document in the
 xref:../install_config/configuring_sdn.adoc#configuring-the-pod-network-on-masters[Configuring
 SDN documentation].
 
-If you would like to plan for a larger environment, here are some example values to consider adding to the `[OSE3:vars]` section in your
-Ansible inventory file:
+To plan for a larger environment, the following are suggested values to consider
+adding to the `[OSE3:vars]` section in your Ansible inventory file:
```

Comment:
Sorry @jeremyeder  I meant in your initial PR:

https://github.com/openshift/openshift-docs/pull/4171/files#diff-63cc6ae3cf2d348826dbb2e7f6cb0b35R132